manifests/s/Samsung/GalaxyBudsManagerExe/2.1.220907.51/Samsung.GalaxyBudsManagerExe.locale.en-US.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.GalaxyBudsManager
+PackageVersion: 2.1.220907.51
+PackageLocale: en-US
+Publisher: Samsung Electronics Co, Ltd.
+PublisherUrl: https://www.samsung.com/
+PublisherSupportUrl: https://www.samsung.com/ca/support/
+PrivacyUrl:   https://www.samsung.com/ca/sustainability/security-and-privacy/
+PackageName: Galaxy Buds Manager
+PackageUrl: https://www.samsung.com/ca/support/mobile-devices/galaxy-buds-install-galaxy-wearable/
+License: Proprietary
+Copyright: Copyright © 2019 Samsung Electronics Co., Ltd. All rights reserved.
+ShortDescription: GalaxyBudsManager 2.1.220907.51

```

Comment:
Please provide a better description of what GalaxyBudsManager is used for / does